controllers/common/common.go

```diff
@@ -92,6 +95,48 @@ func IsPbForPoicySet(pb *policiesv1.PlacementBinding) bool {
 	return found
 }
 
+// PoliciesInPlacementBinding returns a list of the Policies that are either direct subjects of
+// the given PlacementBinding, or are in PolicySets that are subjects of the PlacementBinding.
+// The list items are not guaranteed to be unique (for example if a policy is in multiple sets).
+func PoliciesInPlacementBinding(
```

Comment:
Get + policiesInPlacemantBiding?

Reply:
Renamed to `GetPoliciesInPlacementBinding`